File: credito.py
#Kleber Jesus Montes Compañ
#Ingenieria en Animacion Digital
#ID: 00623774 """
print("Digite el plazo otorgado: ")
plazo = int(input())

#asignar el articulo basado en el plazo
if plazo == 48:
    Art = "un computador"
elif plazo == 36:
    Art = "un TV de 42 pulgadas"
elif plazo == 24:
    Art = "un BlackBerry"
elif plazo == 12:
    Art = "una camara digital"
elif plazo == 6:
    Art = "una impresora"
else:
    Art = ""  #plazo invalido

#verificar y mostrar resultado
if Art != "":
    print(f"Podria llevar {Art}")
else:
    print("Error. Plazo no existe.")

File: credito.cpp
//Kleber Jesus Montes Compañ
//Ing Animacion digital
//00623774
#include <iostream>
#include <cstdlib>
#include <string>
using namespace std;

int main () {

    string Art;
    int plazo;
cout << "Digite el plazo otorgado: " << endl;
cin >> plazo;
switch (plazo) {
    case 48: Art = "un computador."; break;
    case 36: Art = "una TV de 42 pulgadas."; break;
    case 24: Art = "un BlackBerry."; break;
    case 12: Art = "una camara digital."; break;
    case 6: Art = "una impresora."; break;
    default: Art = "."; break;
}
if (Art == ".") {
    cout << "Error. Plazo no existe." << endl;
} else {
    cout << "Podria llevar " << Art << endl;
}
return 0;
}